expense-splitter/src/components/expense/CreateExpense.jsx

```diff
@@ -106,59 +133,97 @@ export default function CreateExpense() {
     return (
       <div
         key={friend.name}
-        className="mb-2 flex items-center justify-between gap-2"
+        className="mb-2 grid grid-cols-3 items-center justify-between gap-2"
       >
         <label className="mr-2">{friend.name}</label>
-        <input
-          className="ml-auto w-[60px] text-center"
-          name={friend.name}
-          placeholder="0"
-          defaultValue={0}
-          {...register(`${friend.name}`, {
-            pattern: {
-              value: /^[0-9]{1,2}$/i,
-              message: "invalid type, please enter a number between 1-100%",
-            },
-          })}
-        />
-        <div className="field w-28 text-center">{friend.dollar}</div>
+        <div className="relative ml-auto flex w-[68px] items-center">
+          <input
+            className="w-full pr-4"
+            name={friend.name}
+            required
+            placeholder="0"
+            defaultValue={0}
```

Comment:
I wonder if there is a way to avoid having a default value so that when the user clicks on the input field they don't have to delete the 0 in order to input their value.

I'm sure most of your calculations are based on the 0 being there so I know it's a big undertaking.

Reply:
Yes another case I didn't want to get into in this round. It would be nice though. Let's see if there's time.